README.md

```diff
@@ -44,7 +43,7 @@ By default `ON`, set it to `OFF` if you do not want to update the already existi
 
 3. `CMAKE_PREFIX_PATH`
 
-Required for installing. Defines the path where the library is to be installed. Set it to, `/usr/local/include` on Linux based systems.
+Required for installing. Defines the path where the library is to be installed. Set it to, `/usr/local/include` on Linux based syst 
```

Comment:
```suggestion
Required for installing. Defines the path where the library is to be installed. Set it to, `/usr/local/include` on Linux based systems.
```

Reply:
The README on your system is outdated. Please update your master,

```
git remote set-url origin https://github.com/codezonediitj/adaboost
git pull origin master
```
Then make the updates.